### DISCUSSION TITLE
LonaView versus View

### DISCUSSION THREAD

**Discussion by metaperl** [lona-web-org/lona#460] [Q&A] [answered]
In [this sample code](https://lona-web.org/1.x/tutorial/01-getting-started/index.html#hello-world) we see `lona.View` being subclassed. In this [other sample code](https://lona-web.org/1.x/basic-concept.html#workflow) we see `lona.view.LonaView` being subclassed. Why would one pick one over the other?

**Comment by fscherf** [comment, marked as answer]
Hi @metaperl,
`LonaView` and `View`, or `LonaApp` and `App` are actually just aliases for each other that are only there for compatibility reasons. In the day there were inconsistencies in the API like `lona.routing.Route` vs `lona.view.LonaView`, and I decided to ditch the `Lona` prefix since this is the name of the module to import from anyway.

All classes, starting with `Lona` will be removed in Lona 2. If the old names are used in the documentation, that's just legacy and should be removed.
